src/app/verification-addresses/features/new-iva-dialog/new-iva-dialog.component.ts

```diff
@@ -48,10 +48,10 @@ export class NewIvaDialogComponent {
    * Value prompts for the different IVA types
    */
   valuePrompts: { [key in keyof typeof IvaType]: string } = {
-    Phone: 'Please enter your phone number to receive SMS messages:',
+    Phone: 'Please enter your mobile phone number:',
```

Comment:
Most stationary phones can also receive SMS, maybe you want to use your office phone number.

Reply:
The tooltip is too long otherwise (it changes the width of the dialog, which looks unpleasant). SMS-enabled phone number could be an alternative.

Reply:
How about 'Please enter your phone number to receive an SMS:' ?